### ISSUE TITLE
Make improvements to screen-reader behaviour

### ISSUE THREAD

**Issue by simonmitchell** [3sidedcube/ThunderCloud#231]
## Expected Behaviour

- ToggleableListItem should read as "Collapsable, Collapsed" e.t.c. based on it's state.
- App collection list should read out the "Installed" state of the app.
- Radio buttons should be read with role and selected state.
- Number of selected answers in quiz should be announced with context: '0 out of 1 possible answers selected'.
- Skip image area selection questions with announcement: 'This question cannot be completed with VoiceOver enabled, please navigate to next question'.
- Quiz slider position is announced as percentage rather than the value displayed above.

## Current Behaviour

None of the above happens

## Possible Solution

Make the fixes!

## Context

Need to make these changes to be ADA compliant

## Your Environment

* ThunderCloud version: 1.6.2
* iOS / macOS version: All
* Link to your project, if public: N/A
